src/raftstore/store/metrics.rs

```diff
@@ -213,10 +213,19 @@ lazy_static! {
             "Bucketed histogram of rocksdb ingestion durations",
             exponential_buckets(0.005, 2.0, 20).unwrap()
         ).unwrap();
+
     pub static ref RAFT_INVALID_PROPOSAL_COUNTER_VEC: IntCounterVec =
         register_int_counter_vec!(
             "tikv_raftstore_raft_invalid_proposal_total",
             "Total number of raft invalid proposal.",
             &["type"]
         ).unwrap();
+
+    pub static ref RAFT_EVENT_TIMEOUT: HistogramVec =
+        register_histogram_vec!(
+            "tikv_raftstore_event_timeout",
+            "Raft store each event timeout",
+            &["type"],
+            exponential_buckets(0.001, 2.0, 20).unwrap()
```

Comment:
`0.001` may be too large. The smallest bucket is 1ms and most events executes less than 1ms. What do you think @huachaohuang ?

Reply:
It may be fine because if the event takes less than 1ms, we don't care it anyway.